usr/share/rear/conf/GNU/Linux.conf

```diff
@@ -1,7 +1,12 @@
 # Architecture-independend Files
 REQUIRED_PROGS=(
 "${REQUIRED_PROGS[@]:-}"
+chroot
+ip
+less
 parted
+route
```

Comment:
This broke ReaR on some CentOS installs (like the CentOS 7 Vagrant image). `route` is not installed every time nowadays, as it is considered legacy. ReaR seems to use properly `ip route` everywhere, so `route should not be needed and should stay in `PROGS`.

Reply:
Fixed via https://github.com/rear/rear/pull/2297